metadata/com.sparklinlabs.Superpowers.appdata.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>com.sparklinlabs.Superpowers.desktop</id>
```

Comment:
Yes, I know its valid, ran the same thing myself on other flatpaks, which ended up missing the `name` field. Its a bit odd.